controllers/machinehealthcheck_controller.go

```diff
@@ -57,6 +57,9 @@ const (
 	// EventRemediationRestricted is emitted in case when machine remediation
 	// is restricted by remediation circuit shorting logic.
 	EventRemediationRestricted string = "RemediationRestricted"
+
+	oldEmrAnnotationKey = "sigs.k8s.io/cluster-api/old-emr-flag"
+	oldEmrAlertTimeout  = time.Hour * 48
```

Comment:
Where this timeout comes from? does it applies to all the EMR implementations?

Reply:
> Where this timeout comes from? does it applies to all the EMR implementations?

This change originated from [this](https://github.com/openshift/enhancements/pull/551#issuecomment-812138136) discussion.
Yes in it's current form it applies to all EMR implementations

Reply:
I personally not consider slack as a place to use for documenting requirements or design decisions, so I'm +1 to keep track of context in the MHC proposal (KEP are living documents) or in a code comment, otherwise people will get lost

Reply:
> I personally not consider slack as a place to use for documenting requirements or design decisions, so I'm +1 to keep track of context in the MHC proposal (KEP are living documents) or in a code comment, otherwise people will get lost

Agreed :+1: